lib/tasks/sdk.rake

```diff
@@ -48,6 +48,26 @@ task 'clean_env' do
   end
 end
 
+desc 'Wipe integration'
+task 'wipe', :option do |_, args|
+  flavor = args[:option] || false
+  abort "please specify an integration to remove" if not flavor
+  check_env
+  input = ''
+  print "Are you sure you want to remove the integration (y/n)? "
+  input = STDIN.gets.chomp
+  case input.upcase
+  when "Y"
+    `rm -rf #{ENV['SDK_HOME']}/#{flavor}` if File.directory?("#{ENV['SDK_HOME']}/#{flavor}")
+    `rm -rf #{ENV['SDK_HOME']}/ci/#{flavor}.rake` if File.exist?("#{ENV['SDK_HOME']}/ci/#{flavor}.rake")
+    puts "source and CI files."
+    sh "sed -i '' \"/#{flavor}/d\" #{ENV['SDK_HOME']}/.travis.yml"
+    sh "sed -i '' \"/#{flavor}/d\" #{ENV['SDK_HOME']}/circle.yml"
```

Comment:
The brackets won't work on BSD sed... But I totally agree on the input so I've changed the calls on sed (and wrapped them in the helper function) so that we only remove exact hits in the travis and circle yamls.